### ISSUE TITLE
DataTable: Selection - Checkboxes inaccessible for voice commands

### ISSUE THREAD

**Issue by MarkusChur** [primefaces/primefaces#5150]
## Summary
When using selection for DataTable, checkboxes can't be toggled via voice commands. The root cause is very similar to [ISSUE-4979](https://github.com/primefaces/primefaces/issues/4979) which is fixed by [PULL-4981](https://github.com/primefaces/primefaces/pull/4981).

* The `<div>` that represents the checkbox doesn't have a role attribute
* The `<div>` that represents the checkbox doesn't have aria-checked attribute
* The `<input>` is not required and can be removed if the `<div>` is fixed

## 1) Environment
*  Does not work with 7.0.6
*  Does not work in the showcase [DataTable - Selection](https://www.primefaces.org/showcase/ui/data/datatable/selection.xhtml)

## 2) Expected behavior
Checkbox is listed when calling a command to interact with checkboxes

> Click checkbox

## 3) Actual behavior
UI Elements are not recognized as checkboxes, therefore nothing happens. You are not able to interact with checkboxes

## 4) Steps to reproduce
Try to use voice commands to access the components checkboxes

## 5) Sample Fix
* Please refer to [PULL-4981](https://github.com/primefaces/primefaces/pull/4981)

**Comment by melloware** [comment]
Any chance you want to submit a PR for this?

**Comment by MarkusChur** [comment]
yes but not before next week I guess :)
